### PR TITLE
Mark missed translations

### PR DESCRIPTION
**Pull request by MichaelChirico**
Follow-up to #5875 -- earmarked the few messages remaining that need translation still. One of them was missed when finding fuzzy translations.

@hongyuanjia feel free to either repeat the last PR (making edits locally + pushing to a separate PR) or to add suggested changes here in the review UI. Whatever's easiest for you. Thanks again!

### CODE REVIEW

File: po/R-zh_CN.po [hunk 1]
```diff
@@ -1848,7 +1848,7 @@ msgstr "data.table %s 的 '%s' 列中存在缺失值：'%s'。若使 foverlaps()
 msgid ""
 "All entries in column '%s' should be <= corresponding entries in column '%s' "
 "in data.table x."
-msgstr "应该<=列中相应的条目"
+msgstr "<needs update>应该<=列中相应的条目"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:1851]: `列 '%s' 中所有条目应该 <= data.table x 中列 '%s' 中对应的条目。`

File: po/R-zh_CN.po [hunk 2]
```diff
@@ -2698,17 +2698,17 @@ msgstr "'x' 必须是一个 list"
 msgid ""
 "Character objects provided in the input are not scalar objects, if you need "
 "them as character vector rather than a name, then wrap each into 'I' call: %s"
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:2701]:
```
msgstr ""
"输入中所提供的字符串对象不是标量(长度为 1 )对象，如果您需要它们作为字符串向量而"
"不是名称，请将每个对象放入到 'I' 调用中：%s"
```

File: po/R-zh_CN.po [hunk 3]
```diff
@@ -2698,17 +2698,17 @@ msgstr "'x' 必须是一个 list"
...
 
 #: programming.R:50
 #, c-format
 msgid "'env' must not be missing"
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:2706]:
```
msgstr "必须提供 'env' 参数"
```

File: po/R-zh_CN.po [hunk 4]
```diff
@@ -2698,17 +2698,17 @@ msgstr "'x' 必须是一个 list"
...
 
 #: programming.R:56
 #, c-format
 msgid "'env' must be a list or an environment"
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:2711]:
```
msgstr "'env' 必须为一列表或环境"
```

File: po/R-zh_CN.po [hunk 5]
```diff
@@ -2718,7 +2718,7 @@ msgstr "参数 'env' 没有名"
 #: programming.R:65
 #, c-format
 msgid "'env' argument has zero char names"
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:2721]:
```
msgstr "'env' 参数的名字为零字符长度（即\"\"）"
```

File: po/R-zh_CN.po [hunk 6]
```diff
@@ -2991,17 +2991,17 @@ msgstr "内部错误：ncol(current)==ncol(target) 之前已经检查"
 #: setops.R:159 setops.R:170
 #, c-format
 msgid "Datasets have different %s. 'target': %s. 'current': %s."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:2994]:
```
msgstr "输入的数据拥有不同的%s。'target'：%s，'current'：%s。"
```

File: po/R-zh_CN.po [hunk 7]
```diff
@@ -2991,17 +2991,17 @@ msgstr "内部错误：ncol(current)==ncol(target) 之前已经检查"
...
 
 #: setops.R:161 setops.R:162
 #, c-format
 msgid "has no key"
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:2999]:
```
msgstr "没有键列"
```

File: po/R-zh_CN.po [hunk 8]
```diff
@@ -2991,17 +2991,17 @@ msgstr "内部错误：ncol(current)==ncol(target) 之前已经检查"
...
 
 #: setops.R:172 setops.R:173
 #, c-format
 msgid "has no index"
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3004]:
```
msgstr "没有索引"
```

File: po/R-zh_CN.po [hunk 9]
```diff
@@ -3047,11 +3047,11 @@ msgstr "内部错误：此时不匹配的因子类型应已被发现"
 
 #: shift.R:3
 msgid "Provided argument fill="
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3050]:
```
msgstr "提供的 fill= 参数"
```

File: po/R-zh_CN.po [hunk 10]
```diff
@@ -3047,11 +3047,11 @@ msgstr "内部错误：此时不匹配的因子类型应已被发现"
...
 
 #: shift.R:3
 msgid "will be ignored since type='shift'."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3054]:
```
msgstr "将被忽略，因为 type='shift'。"
```

File: po/R-zh_CN.po [hunk 11]
```diff
@@ -3079,14 +3079,14 @@ msgstr "%3$s 中 %1$s 也 %2$s 不存在"
 
 #: test.data.table.R:114
 msgid "object '%s' not found"
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3082]:
```
msgstr "未找到 '%s' 对象"
```

File: po/R-zh_CN.po [hunk 12]
```diff
@@ -3079,14 +3079,14 @@ msgstr "%3$s 中 %1$s 也 %2$s 不存在"
...
 
 #: test.data.table.R:138
 #, c-format
 msgid ""
 "memtest intended for Linux. Step through data.table:::rss() to see what went "
 "wrong."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3089]:
```
msgstr "memtest 仅适用于 Linux。请逐步执行 data.table:::rss() 以查看错误原因。"
```

File: po/R-zh_CN.po [hunk 13]
```diff
@@ -3138,7 +3138,7 @@ msgstr "'keep'所含整数值应在 %d 和 %d 间"
 #: transpose.R:51
 #, c-format
 msgid "The argument 'type.convert' does not support empty list."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3141]:
```
msgstr "参数 'type.convert' 不支持空列表。"
```

File: po/R-zh_CN.po [hunk 14]
```diff
@@ -3148,15 +3148,15 @@ msgid ""
 "element is not allowed unless all elements are functions with length equal "
 "to %d (the length of the transpose list or 'keep' argument if it is "
 "specified)."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3151]:
```
msgstr ""
"当参数 'type.convert' 包含一个未命名元素时，此元素应为最后一个元素，且应为一"
"个函数。不允许包含多个未命名元素，除非所有元素都为函数，且所有元素的总个数为"
"%d（即输入列表转置后的长度或 'keep' 参数指定的长度）。"
```

File: po/R-zh_CN.po [hunk 15]
```diff
@@ -3148,15 +3148,15 @@ msgid ""
...
 
 #: transpose.R:66
 #, c-format
 msgid ""
 "When the argument 'type.convert' contains transpose list indices, it should "
 "be a named list of non-missing integer values (with no duplicate) except the "
 "last element that should be unnamed if it is a function."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3159]:
```
msgstr ""
"当参数 'type.convert' 包含转置列表的索引时，它应该为一个命名列表，其中仅包含"
"非缺失的整数值（且无重复）。除非该列表的最后一个元素是一个函数，这时该元素应"
"未命名。"
```

File: po/R-zh_CN.po [hunk 16]
```diff
@@ -3165,23 +3165,23 @@ msgid ""
 "should be integer values contained in the argument 'keep' (if it is "
 "specified) or be between %d and %d (if it is not). But '%s' is/are not "
 "contained in '%s'."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3168]:
```
msgstr ""
"当参数 'type.convert' 包含转置列表的索引时，它们应该是参数 'keep' （如果"
"提供了的话）中的整数值，或者（如果 'keep' 没有指定的话）应介于 %d 和 %d 之间。"
"但 '%s' 并不包含在 '%s' 中。"
```

File: po/R-zh_CN.po [hunk 17]
```diff
@@ -3165,23 +3165,23 @@ msgid ""
...
 
 #: transpose.R:74
 #, c-format
 msgid ""
 "In the argument 'type.convert', '%s' was ignored because all elements in the "
 "transpose list or elements corrisponding to indices specified in the 'keep' "
 "argument have already been converted."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3176]:
```
msgstr ""
"忽略参数 'type.convert' 中的 '%s'，因为转置列表中的所有元素或参数 'keep'"
"中索引对应的元素已经被转换。"
```

File: po/R-zh_CN.po (hunk 1 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:1851]: FYI you can use the 'Suggestion' tool, then in the review UI it's one click to make the change to the file itself:

image

Alternatively, you can put the change in a code block with "language" (\`\`\`suggestion) like so:

```suggestion
msgstr "列 '%s' 中所有条目应该 <= data.table x 中列 '%s' 中对应的条目。"
```

https://haacked.com/archive/2019/06/03/suggested-changes/

File: po/R-zh_CN.po (hunk 2 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:2701]:
```suggestion
msgstr ""
"输入中所提供的字符串对象不是标量(长度为 1 )对象，如果您需要它们作为字符串向量而"
"非名称，请将每个对象放入到 'I' 调用中：%s"
```

File: po/R-zh_CN.po (hunk 3 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:2706]:
```suggestion
msgstr "必须提供 'env' 参数"
```

File: po/R-zh_CN.po (hunk 4 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:2711]:
```suggestion
msgstr "'env' 必须为一列表或环境"
```

File: po/R-zh_CN.po (hunk 5 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:2721]:
```suggestion
msgstr "'env' 参数的名字为零字符长度（即\"\"）"
```

File: po/R-zh_CN.po (hunk 6 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:2994]:
```suggestion
msgstr "输入的数据拥有不同的%s。'target'：%s，'current'：%s。"
```

File: po/R-zh_CN.po (hunk 7 above)
Comment (MichaelChirico) [inline po/R-zh_CN.po:2999]:
```suggestion
msgstr "没有键列"
```

File: po/R-zh_CN.po (hunk 8 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3004]:
```suggestion
msgstr "没有索引"
```

File: po/R-zh_CN.po (hunk 9 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3050]:
```suggestion
msgstr "提供的 fill= 参数"
```

File: po/R-zh_CN.po (hunk 10 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3054]:
```suggestion
msgstr "将被忽略，因为 type='shift'。"
```

File: po/R-zh_CN.po (hunk 11 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3082]:
```suggestion
msgstr "未找到 '%s' 对象"
```

File: po/R-zh_CN.po (hunk 12 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3089]:
```suggestion
msgstr "memtest 仅适用于 Linux。请逐步执行 data.table:::rss() 以查看错误原因。"
```

File: po/R-zh_CN.po (hunk 14 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3151]:
```suggestion
msgstr ""
"当参数 'type.convert' 包含一个未命名元素时，此元素应为最后一个元素，且应为一"
"个函数。不允许包含多个未命名元素，除非所有元素都为函数，且所有元素的总个数为"
"%d（即输入列表转置后的长度或 'keep' 参数指定的长度）。"
```

File: po/R-zh_CN.po (hunk 15 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3159]:
```suggestion
msgstr ""
"当参数 'type.convert' 包含转置列表的索引时，它应该为一个命名列表，其中仅包含"
"非缺失的整数值（且无重复）。除非该列表的最后一个元素是一个函数，这时该元素应"
"未命名。"
```

File: po/R-zh_CN.po (hunk 16 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3168]:
```suggestion
msgstr ""
"当参数 'type.convert' 包含转置列表的索引时，它们应该是参数 'keep' （如果"
"提供了的话）中的整数值，或者（如果 'keep' 没有指定的话）应介于 %d 和 %d 之间。"
"但 '%s' 并不包含在 '%s' 中。"
```

File: po/R-zh_CN.po (hunk 17 above)
Comment (MichaelChirico) [reply po/R-zh_CN.po:3176]:
```suggestion
msgstr ""
"忽略参数 'type.convert' 中的 '%s'，因为转置列表中的所有元素或参数 'keep'"
"中索引对应的元素已经被转换。"
```

File: po/R-zh_CN.po [hunk 18]
```diff
@@ -3165,23 +3165,23 @@ msgid ""
...
 
 #: transpose.R:83
 #, c-format
 msgid ""
 "The argument 'type.convert' should be TRUE/FALSE, a function, a list of "
 "functions, or a named list of pairs 'fun=indices' with optionally one "
 "unnamed element (a function) but an object of type '%s' was provided."
-msgstr ""
+msgstr "<needs translation>"
```
Comment (hongyuanjia) [inline po/R-zh_CN.po:3184]:
```
msgstr ""
msgstr ""
"参数 'type.convert' 应为 TRUE/FALSE、一个函数、一个包含函数的列表，或一个包含"
"元素 'fun=indices' (函数=索引) 的命名列表，其中可以有一个未命名元素（必须为一"
"个函数），但当前输入中提供了一个类型为 '%s' 的对象。"
```

Comment (MichaelChirico) [reply po/R-zh_CN.po:3197]:
```suggestion
msgstr ""
"参数 'type.convert' 应为 TRUE/FALSE、一个函数、一个包含函数的列表，或一个包含"
"元素 'fun=indices' (函数=索引) 的命名列表，其中可以有一个未命名元素（必须为一"
"个函数），但当前输入中提供了一个类型为 '%s' 的对象。"
```

**Review by MichaelChirico — COMMENTED** [review]
Thanks again!

**Comment by hongyuanjia** [comment]
@MichaelChirico, I have just completed all the missing translations. Those messages in `transpose.R` are challenging to translate.

**Pull request merged**